style.css

```diff
@@ -70,3 +71,250 @@ body {
   list-style: none;
   padding: 0;
 }
+
+.scroll {
+  display: flex;
+  flex-flow: column;
+  justify-content: center;
+  align-items: center;
+  gap: 30px;
+  width: 100%;
+  padding: 100px 0;
+  background-color: #f7f7f9;
+}
+
+.box-contain {
+  border: 1px solid #7a869a;
+  width: 90%;
+  margin: 0 auto;
+  border-radius: 16px;
+  background-color: #fff;
+  list-style: none;
+  margin-bottom: 50px;
+}
+
+.box-contain ul {
+  padding: 0;
+  margin: 0;
+}
+
+.ptflio-item {
+  width: 90%;
+  margin: 0 auto;
+  list-style: none;
+  padding: 20px 0;
+  background-color: #fff;
+  display: flex;
+  flex-flow: column;
+}
+
+.ptflio-image {
+  height: 220px;
+  width: 100%;
+  left: 16px;
+  top: 16px;
+  border-radius: 8px;
+}
+
+.lists-container {
+  width: 100%;
+  align-items: center;
+}
+
+.lists {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+}
+
+.lists-item {
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 15px;
+  text-transform: capitalize;
+}
+
+.lists .obect1 {
+  text-transform: capitalize;
+  color: #344563;
+}
+
+.text-holder {
+  position: static;
+  width: 327px;
+  left: 24px;
+  top: 276px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding-top: 8px;
+  padding-bottom: 6px;
+}
+
+.helloo {
+  position: static;
+  width: 327px;
+  height: 146px;
+  left: 24px;
+  top: 276px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  padding-top: 8px;
+  padding-bottom: 6px;
+}
+
+.language-container {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  list-style: none;
+}
+
+.language-item {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 5px 10px;
+  margin: 0 2px;
+  font-size: 14px;
+  border-color: #396df2;
+  color: #396df2;
+}
+
+.butto {
+  font-size: 19px;
+  color: #396df2;
+  padding: 13px;
+  background-color: transparent;
+  margin-top: 2rem;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+}
+
+.about-me {
+  width: 90%;
+  margin: 0 auto;
+  overflow: hidden;
+  background-color: #fff;
+  border-top-right-radius: 5rem;
+  padding-top: 5rem;
```

Comment:
Please ensure that the `about section` content is aligned with flexbox according to the project requirement.